Migration.md

```diff
@@ -5,13 +5,16 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
-## Ignition Common 4.X to 5.X
+## Gazebo Common 4.X to 5.X
 
 ### Deprecations
 
 1. `Submesh::MaterialIndex` is deprecated. `SubMesh::GetMaterialIndex` should
    be used instead, which properly handles submeshes having no material index
    applied to them.
+2. The `ignition` namespace is deprecated and will be removed in future versions.  Use `gz` instead.
+3. Header files under `ignition/...` are deprecated and will be removed in future versions.
+   Use `gz/...` instead.
```

Comment:
We should add a note about the console deprecations (`ignerr` etc)

Reply:
Done! https://github.com/gazebosim/gz-common/pull/356/commits/dcc40dce6630288ba9ed67b08a0bd08499be3404